docs/backupstoragelocation-definition.md

```diff
@@ -0,0 +1,65 @@
+# Ark Backup Storage Locations
+
+## Backup Storage Location
+
+Ark can store backups in a number of locations. These are represented in the cluster via the `BackupStorageLocation` CRD.
+
+Ark needs a minimum of one `BackupStorageLocation`, named `default`, in order to store backups. Backups that do not explicitly specify a storage location will be saved to the `default` `BackupStorageLocation`.
```

Comment:
Probably worth noting that by default, the default location is named "default", but you can override the default name with a server flag.  Good luck wording that :)

Reply:
Here's a few different wordings that I came up with:

1. Ark needs a minimum of one `BackupStorageLocation`. Without any changes, this will be expected to be named `default`, however  the expected name can be changed with the `--default-backup-storage-location` flag on the `ark server` command. Backups that do not explicitly specify a storage location will be saved to this `BackupStorageLocation`.


1. Ark must have a minimum of one `BackupStorageLocation`. By default, Ark expects this to be named `default`. The name can be changed by specifying `--default-backup-storage-location` on `ark server`. Backups that do not explicitly specify a storage location will be saved to this `BackupStorageLocation`.


1. Ark must have at least one `BackupStorageLocation`. By default, this is expected to be named `default`, however the name can be changed by specifying `--default-backup-storage-location` on `ark server`.  Backups that do not explicitly specify a storage location will be saved to this `BackupStorageLocation`.

Reply:
I like (3)